### ISSUE TITLE
[css-layout-api] Should display: layout(x) work on any element?

### ISSUE THREAD

**Issue by zcorpan** [w3c/css-houdini-drafts#796]
See https://github.com/whatwg/html/issues/3915 where I wonder if layout() should work on `fieldset`

**Comment by zcorpan** [comment]
cc @bfgeek

**Comment by AmeliaBR** [comment]
Alternative question:

Can `display: layout(-browser-internal-function)` be used to finally specify `fieldset` and `legend` behavior in standard CSS syntax?

**Comment by zcorpan** [comment]
@AmeliaBR no, web compat requires magic if author CSS changes 'display'. But I have a different proposal at https://github.com/whatwg/html/issues/3912
